java/src/main/java/org/signal/libsignal/metadata/SealedSessionCipher.java

```diff
@@ -49,102 +48,87 @@
 
   private final SignalProtocolStore signalProtocolStore;
-  private final String              localE164Address;
-  private final String              localUuidAddress;
-  private final int                 localDeviceId;
+  private final int localDeviceId;
 
   public SealedSessionCipher(SignalProtocolStore signalProtocolStore,
-                             UUID localUuid,
-                             String localE164Address,
-                             int localDeviceId)
-  {
+                             int localDeviceId) {
     this.signalProtocolStore = signalProtocolStore;
-    this.localUuidAddress    = localUuid != null ? localUuid.toString() : null;
-    this.localE164Address    = localE164Address;
-    this.localDeviceId       = localDeviceId;
+    this.localDeviceId = localDeviceId;
   }
 
-  public byte[] encrypt(SignalProtocolAddress destinationAddress, SenderCertificate senderCertificate, byte[] paddedPlaintext)
-      throws InvalidKeyException, UntrustedIdentityException
-  {
-    CiphertextMessage message       = new SessionCipher(signalProtocolStore, destinationAddress).encrypt(paddedPlaintext);
-    IdentityKeyPair   ourIdentity   = signalProtocolStore.getIdentityKeyPair();
-    ECPublicKey       theirIdentity = signalProtocolStore.getIdentity(destinationAddress).getPublicKey();
+  public byte[] encrypt(SignalProtocolAddress destinationAddress, byte[] paddedPlaintext)
+          throws InvalidKeyException, UntrustedIdentityException {
+    return encrypt(signalProtocolStore.getIdentityKeyPair(), destinationAddress, paddedPlaintext);
+  }
+
+  byte[] encrypt(IdentityKeyPair ourIdentity, SignalProtocolAddress destinationAddress, byte[] paddedPlaintext)
+          throws InvalidKeyException, UntrustedIdentityException {
+    CiphertextMessage message = new SessionCipher(signalProtocolStore, destinationAddress).encrypt(paddedPlaintext);
+    ECPublicKey theirIdentity = signalProtocolStore.getIdentity(destinationAddress).getPublicKey();
 
-    ECKeyPair     ephemeral           = Curve.generateKeyPair();
-    byte[]        ephemeralSalt       = ByteUtil.combine("UnidentifiedDelivery".getBytes(), theirIdentity.serialize(), ephemeral.getPublicKey().serialize());
-    EphemeralKeys ephemeralKeys       = calculateEphemeralKeys(theirIdentity, ephemeral.getPrivateKey(), ephemeralSalt);
-    byte[]        staticKeyCiphertext = encrypt(ephemeralKeys.cipherKey, ephemeralKeys.macKey, ourIdentity.getPublicKey().getPublicKey().serialize());
+    ECKeyPair ephemeral = Curve.generateKeyPair();
+    byte[] ephemeralSalt = ByteUtil.combine("UnidentifiedDelivery".getBytes(), theirIdentity.serialize(), ephemeral.getPublicKey().serialize());
+    EphemeralKeys ephemeralKeys = calculateEphemeralKeys(theirIdentity, ephemeral.getPrivateKey(), ephemeralSalt);
+    byte[] staticKeyCiphertext = encrypt(ephemeralKeys.cipherKey, ephemeralKeys.macKey, ourIdentity.getPublicKey().getPublicKey().serialize());
```

Comment:
Are these changes just formatting?  It seems important to be able to tell what's different between our fork and the original.  Could you try to undo the changes that are only formatting?